e2e/util/logger.go

```diff
@@ -0,0 +1,41 @@
+package util
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+func CreateLogger(executableName string, verbose bool) *zap.SugaredLogger {
```

Comment:
We don't need the executable name or verbose flag.

The executable name is available in os.Args[0], but it may be "go" when we run the tests as `go test ...`, and "ramen-e2e" when we run from the executable. It will be best to support both use cases. If using os.Args[0] does not work, it is fine to hard code the default log name using private constant in this file.

We can add later a --logfile argument and use it here, but lets defer this work so we can have the default log file available sooner. Using custom logfile is nice to have and we can add it when after we finished with more important stuff.

Reply:
Lets use constant log name. Add a private constant in the log.go.

When we add a --logfile option the option will use a default value anyway.